Cue/app/tabs/library/DeckView.js

```diff
@@ -253,23 +285,25 @@ class DeckView extends React.Component {
         <CueHeader
           leftItem={leftItem}
           rightItems={rightItems}
-          key={this.state.isFiltering.toString() + this.state.scrollPosition.toString()}
-          containerStyles={{elevation: this.state.scrollPosition > this.state.headerHeight ? 4 : 0}} />
-        <ScrollView
-          style={{flex: 1}}
-          onScroll={this._onScroll}>
+          key={this.state.filtering.toString()} />
```

Comment:
This sets the key on the header to work around the problem where some items would disappear from the header after the Navigator stack pops.